pipeline-model-json-shaded/pom.xml

```diff
@@ -0,0 +1,104 @@
+<!--
```

Comment:
So all the dependencies shaded in here are directly related to either `json-schema-validator` or `jackson-datatype-json-org` - we're not invoking any of the other shaded dependencies directly anywhere, just the JSON stuff.